tests/test_parse_data.py

```diff
@@ -0,0 +1,92 @@
+from typing import Generator
+from datetime import datetime
+import json
+import os
+import inspect
+import sys
+
+currentdir = os.path.dirname(   # type: ignore
+    os.path.abspath(
+        inspect.getfile(inspect.currentframe())     # type: ignore
+    )
+)
+parentdir = os.path.dirname(currentdir)
+sys.path.insert(0, parentdir) 
+
+from src.parse_data import build_stories_info   # type: ignore
+
+
+def generate_stub_stories() -> Generator[dict, None, None]:
+    for i in range(2):
+        stub_data = {
+                'title': 'Test' + str(i),
+                'score': 1234,
+                'url': 'some-url-here'
+        }
+        yield stub_data
+
+
+def generate_stub_stories_without_url() -> Generator[dict, None, None]:
+    for i in range(2):
+        stub_data = {
+                'title': 'Test' + str(i),
+                'score': 1234,
+                'url': 'there is no link'
+        }
+        yield stub_data
+
+
+def test_build_stories_info():
```

Comment:
Could you please add a doc-string to each test definition, which will highlight steps and the purpose of each test?

Reply:
Okay, I will add it